src/pipeline-server/docker-compose.pipeline-server.yml

```
@@ -14,13 +14,13 @@ services:
   camera-simulator:
     container_name: camera-simulator
     image: aler9/rtsp-simple-server
+    ipc: "none"
     ports:
       - "8555:8554"
     environment:
-      - http_proxy=
-      - https_proxy=
-      - HTTP_PROXY=
-      - HTTPS_PROXY=
+      http_proxy: ${http_proxy}
+      https_proxy: ${https_proxy}
+      no_proxy: ${no_proxy}
```

I found an issue when setting proxies like this. The correct way is:

```suggestion
      - HTTP_PROXY
      - HTTPS_PROXY
      - NO_PROXY
```